NEMO/views/tasks.py

```diff
@@ -67,15 +67,25 @@ def create(request):
     task = form.save()
     task_images = save_task_images(request, task)
 
-    if not settings.ALLOW_CONDITIONAL_URLS and task.force_shutdown:
-        site_title = ApplicationCustomization.get("site_title")
+    save_error = save_task(request, task, task_images)
+
+    if save_error:
         dictionary = {
             "title": "Task creation failed",
             "heading": "Something went wrong while reporting the problem",
-            "content": f"Tool control is only available on campus. When creating a task, you can't force a tool shutdown while using {site_title} off campus.",
+            "content": save_error,
         }
         return render(request, "acknowledgement.html", dictionary)
 
+    return redirect("tool_control")
+
+
+def save_task(request, task: Task, task_images: List[TaskImages]):
```

Comment:
```suggestion
def save_task(request, task: Task, task_images: List[TaskImages]=None):
```

Reply:
I would change this to actually save the task, as its name suggests (by passing the form, like in the save comment method).
for the kiosk estimated resolution, I believe you can set it by using `form.instance.estimated_resolution_time = estimated_resolution_time` before calling `save_task()`

Reply:
I changed this slightly.  Updating the form value directly didn't work from my tests.  I put the task.save() inside the method but did not pass the entire form as it's also needed for the save_task_images method.